conf/ExtraPackageConfigVars.cmake.in

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+set(NAME "ICUB")
+
+
+set(ICUB_IS_SHARED_LIBRARY @BUILD_SHARED_LIBS@)
+
+set(ICUB_C_COMPILER_ID "@CMAKE_C_COMPILER_ID@")
+set(ICUB_C_COMPILER_VERSION "@CMAKE_C_COMPILER_VERSION@")
+set(ICUB_C_FLAGS "@CMAKE_C_FLAGS@")
+
+set(ICUB_CXX_COMPILER_ID "@CMAKE_CXX_COMPILER_ID@")
+set(ICUB_CXX_COMPILER_VERSION "@CMAKE_CXX_COMPILER_VERSION@")
+set(ICUB_CXX_FLAGS "@CMAKE_CXX_FLAGS@")
+
+set(ICUB_MODULE_PATH "@ICUB_MODULE_PATH@" CACHE INTERNAL "iCub cmake scripts directory")
+set(ICUB_LIBRARIES "@ICUB_TARGETS@" CACHE INTERNAL "List of iCub libraries")
+set(ICUB_LINK_FLAGS "@ICUB_LINK_FLAGS@" CACHE INTERNAL "List of iCub linker options")
+set(ICUB_LINK_DIRECTORIES "@ICUB_LINK_DIRECTORIES@")
+set(ICUB_INSTALL_PREFIX "@CMAKE_INSTALL_PREFIX@")
```

Comment:
Are you sure that this is relocatable? To check, open the `ICUBConfig.cmake` file to check this line. Based on https://cmake.org/cmake/help/v3.4/module/CMakePackageConfigHelpers.html#generating-a-package-configuration-file and https://github.com/robotology/ycm/blob/master/modules/InstallBasicPackageFiles.cmake#L657 , I would exect that you need to use  `@PACKAGE_CMAKE_INSTALL_PREFIX@` instead of  `@CMAKE_INSTALL_PREFIX@`.